src/shapes/Object/Object.ts

```diff
@@ -610,12 +610,20 @@ export class FabricObject<
    */
   declare _transformDone?: boolean;
 
+  private static ownDefaults = fabricObjectDefaultValues;
+
+  static get defaultValues() :Record<string, any> {
```

Comment:
why not a method?
I prefer a method and I thought you were against getter/setter.
A method allows computation if needed

Reply:
this is a getter just because it started with a static default values, first then pivoted to a getter ( That allows computation too as far as i see ).
I m in general not against getters, i just don't ant to add them if i don't really need them.
But here to avoid using the prototype we need somehow to replicate the ladder of the prototypes anyway, and this is the simplest thing i could come up with.

Reply:
OK
`getDefaultValue() {}` can do the job 
I am for it, decide as you see fit

Reply:
yes a normal method i think is better.
.defaultValues or .getDefaultValues() is never used anyway from the developer if not for custom checks.
I'll change to a function.

Reply:
> I think `getDefaultValues` should be static. No reason for it to live on the instance. Is there any reason. Default values is a static concept and after initialization no one will use it which is basically the definition/usage of static

i went a couple of time between static and not static and it doesn't make any difference. in general i use static functions when i know i will need them outside the instance ( like fromObject or fromElement ) but for this i don't see any need to be static.

Reply:
@ShaMan123 for some weird reason github answered your question here.